packages/accordions/src/styled/accordion/StyledHeader.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.header';
+
+export interface IStyledHeader {
+  label?: string;
+  isFocused?: boolean;
+  isCompact?: boolean;
+  isExpanded?: boolean;
+}
+
+export const StyledHeader = styled.div.attrs<IStyledHeader>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledHeader>`
+  display: flex;
+  align-items: center;
+  box-shadow: ${props =>
```

Comment:
Should focus be inset here? It's not often that Garden allows focus to extend beyond border boundaries. See https://garden.zendesk.com/react-components/tables/#selectable or https://garden.zendesk.com/react-components/tabs/#default-usage for examples. /cc @steue

Reply:
Why was this resolved?

Reply:
Sorry about that, I was using these comments as a checklist and had resolved it on locally. I'll need to be careful about marking these as resolved if the commits aren't pushed up since it can be confusing.

I pushed the commit up now and marked this as resolved. Feel free to re-open this if further adjustments are needed.